frontend/src/Pages/Dashboard.tsx

```diff
@@ -1,7 +1,8 @@
-import Navbar from '@/components/common/Navbar';
-import Profile from '@/components/common/Profile';
-import UserProfile from '@/components/user.png';
-import users from '@/dummydata';
+import Navbar from '../components/common/Navbar';
+import Profile from '../components/common/Profile';
+import UserProfile from '../components/user.png';
+import users from '../dummydata';
+import { Link } from 'react-router-dom';
```

Comment:
Is there a reason you changed from using the alias to relative path for the import?

Reply:
@RDLegs alias path sir is implemented on this PR , [[SELS-TASK]-[FE]-Mockup Results Page](https://github.com/framgia/sph-els-mark/pull/16 )

Reply:
is it okay that I will update the alias path of this PR after I merge this PR , [[SELS-TASK]-[FE]-Mockup Results Page ](https://github.com/framgia/sph-els-mark/pull/16) ?

Reply:
@Mjdelima20 I think it is okay.

In that case, please disregard this comment for now